packages/xod-client/src/user/actions.js

```diff
@@ -1,13 +1,65 @@
-import { getCompileLimitUrl } from '../utils/urls';
-import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+import 'url-search-params-polyfill';
+
+import {
+  getCompileLimitUrl,
+  getWhoamiUrl,
+  getLoginUrl,
+  getLogoutUrl,
+} from '../utils/urls';
+import * as ActionTypes from './actionTypes';
 
 export const updateCompileLimit = (startup = false) => dispatch =>
   fetch(getCompileLimitUrl(), {
     headers: startup ? { 'x-launch': 'true' } : {},
   }).then(res => (res.ok ? res.json() : null))
     .catch(() => null)
     .then(limit => dispatch({
-      type: UPDATE_COMPILE_LIMIT,
+      type: ActionTypes.UPDATE_COMPILE_LIMIT,
       payload: limit,
     }));
+
+export const toggleAccountPane = () => ({
+  type: ActionTypes.TOGGLE_ACCOUNT_PANE,
+});
+
+const setGrant = grant => ({
+  type: ActionTypes.SET_AUTH_GRANT,
+  payload: grant,
+});
+
+export const getGrant = () => dispatch =>
```

Comment:
@nkrkv used to tell me "this naming is not clear that a function with side-effects, asynchronous and etc. Let's name it with "load" or "fetch" verb. ;)

Reply:
Also, it will be good to add some comments, why we need this one in addition to `login` action.
(now I remember about the renewal of expired tokens and etc, but I can easily forget about it a couple of months later :)

Reply:
@evgenykochetkov Agree with @brusherru for both points. Suggest `getGrant` → `fetchGrant`